Lib/tkinter/__init__.py

```diff
@@ -1045,8 +1045,16 @@ def winfo_reqwidth(self):
         return self.tk.getint(
             self.tk.call('winfo', 'reqwidth', self._w))
     def winfo_rgb(self, color):
-        """Return tuple of decimal values for red, green, blue for
-        COLOR in this widget."""
+        """Return a tuple of the RGB values for color in this widget.
+
+        Args:
+            color: String representation of the color either as a name
```

Comment:
This is not specific to `winfo_rgb()`. All methods that accept color values accept the same format. It is better to define it in the documentation than in the docstring of `winfo_rgb()`.